qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -1,26 +1,50 @@
 #!/bin/sh -ex
 #
-# rbd_mirror_journal.sh - test rbd-mirror daemon in journal-based mirroring mode
+# rbd_mirror.sh - test rbd-mirror daemon in snapshot and journal-based mirroring mode
+#
+# Usage:
+#   rbd_mirror.sh
+#
+# Use environment variable RBD_MIRROR_MODE to set the mode
+# Available modes: snapshot | journal
 #
 # The scripts starts two ("local" and "remote") clusters using mstart.sh script,
 # creates a temporary directory, used for cluster configs, daemon logs, admin
 # socket, temporary files, and launches rbd-mirror daemon.
 #
 
-. $(dirname $0)/rbd_mirror_helpers.sh
+if [ "${#}" -gt 0 ]; then
+    echo "unnecessary arguments: ${@}"
```

Comment:
Since in the rest of the script two-space indentation is used, need to do the same here.